README.md

```diff
@@ -0,0 +1,58 @@
+[![CircleCI](https://circleci.com/gh/HumanCompatibleAI/benchmark-environments.svg?style=svg)](https://circleci.com/gh/HumanCompatibleAI/benchmark-environments) [![codecov](https://codecov.io/gh/HumanCompatibleAI/benchmark-environments/branch/master/graph/badge.svg)](https://codecov.io/gh/HumanCompatibleAI/benchmark-environments) 
+
```

Comment:
Maybe add explanation of what it is? Something like:

> `benchmark-environments` is a suite of benchmarks for imitation-learning and reward-learning algorithms. It is currently a work-in-progress, but we intend for it to eventually contain a suite of diagnostic tasks for reward-learning, wrappers around common RL benchmark environments that help to avoid common pitfalls in implementing imitation learning algorithms (e.g. by masking visible score counters in Gym Atari tasks), and new challenge tasks for imitation- and reward-learning. This benchmark suite is a complement to our [`imitation`](https://github.com/HumanCompatibleAI/imitation) package of baseline algorithms for imitation learning.

Reply:
thanks, agreed a summary would be useful, added an edited version of this